### ISSUE TITLE
Functionality of Popover.Group

### ISSUE THREAD

**Issue by ebuckthal** [CaptainCodeman/svelte-headlessui#22]
Thanks for all the work you've put into this library! Excited to use it and see it grow. 😃 

---

I want to use multiple Popovers to build a more complex site navigation header. Some pseudocode:

```
<script>
   const popover1 = createPopover({});
   const popover2 = createPopover({});
</script>

<button use:popover1.button ... />
<div use:popover1.panel ... />

<button use:popover2.button ... />
<div use:popover2.panel ... />
```

However, if I first click `popover1.button`, then click `popover2.button` (while `popover1.panel` is still "expanded") it will only close `popover1.panel`. I must click `popover2.button` a second time to open `popover2.panel`.

In headlessui, this action is supported with [grouping related popovers](https://headlessui.com/react/popover#grouping-related-popovers) (`<Popover.Group />`).

 Is there a way to achieve that functionality?

**Comment by CaptainCodeman** [comment]
I'll have to have a play around and see what the best way to support that - I'm picturing something a bit like Stripe's top-level nav, is that the right interpretation?

**Comment by ebuckthal** [comment]
Yes, similar to Stripe's navigation. It would be an added bonus if it could support on-hover as well as on-click!

**Comment by slanelb** [comment]
I was looking for the same functionality but have since realised I should be able to achieve it with a single popover being toggled from multiple nav items whilst switching out the content of the popover. I believe Stripe is the same though with more elaborate animations? Though I do not know if this would be enough for accessibility.
Thank you for the great library! Simple to use and I think an elegant solution to the headless UI problem.

**Comment by slanelb** [comment]
I also wanted Popover Group functionality. I've since decided to implement a UI pattern found on [rolex.com](rolex.com). Look at the Menu and Search buttons in the nav. When pressed, all other buttons fade away so no need for grouping even though there are multiple buttons opening the universal dropdown. Intuitively I feel this may be better for accessibility as well as there is only one focussed choice at a time - less opportunity for confusion. 
Just wanted to share it as I now get to use this great library.
